docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -73,10 +74,35 @@ repositories {
 }
 
 dependencies {
-    compileOnly("io.quarkus:quarkus-resteasy:{quarkus-version}")
+    implementation("io.quarkus:quarkus-resteasy:{quarkus-version}")
 }
 ----
 
+== Enable Tests
+
+{project-name} uses Junit5 and to enable it in Gradle we need to add a section to our build file:
+
+[source,groovy,subs=attributes+]
+----
+test {
+  useJUnitPlatform()
+}
+----
+
+To follow up our Rest example from above, we would also need to add two test dependencies:
+
+[source,groovy,subs=attributes+]
+----
+testCompile group: 'io.quarkus', name: 'quarkus-junit5', version: '{quarkus-version}'
+testCompile group: 'io.rest-assured', name: 'rest-assured', version: '{restassured-version}'
+----
+
+Note: {project-name} do not allow both link:getting-started-testing.html[QuarkusTests] and link:getting-started-testing.html#native-executable-testing[SubstrateTests] to run in the same test run.
+SubstrateTests should be seen as integration tests and moved to a different folder
```

Comment:
`SubstrateTest` classes.

I'm wondering about integration tests. Both are kind of integration tests. But well, this may be philosophical.
However, I don't see the need for a different folder. Is it a Gradle thing?

Reply:
Ok... just read the next sentence. It's a gradle thing.

Reply:
yes, gradle will automatically run everything it can find in the test folder.